lib/Doctrine/ORM/NativeQuery.php

```diff
@@ -34,7 +34,7 @@ public function setSQL($sql): self
     /**
      * Gets the SQL query.
      *
-     * @return mixed The built SQL query or an array of all SQL queries.
+     * @return string The built SQL query
```

Comment:
The class is final, and I don't see how we could have anything else than a string here.

Reply:
If you're sure about that, you can add a native return type. Doing that on a `final` class is not a BC break.